lib/screens/editor/versions.dart

```diff
@@ -0,0 +1,398 @@
+                  tag.key,
+                  style: TextStyle(
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.all(6),
+                child:
+                    Text(!tag.value.noChange ? tag.value.oldValue ?? '' : ''),
+              ),
+              Padding(
+                padding: const EdgeInsets.all(6),
+                child: Text(tag.value.newValue ?? ''),
+              )
+            ],
+          ),
+      ],
+    );
+  }
+
+  Card _buildCard(Version version, AppLocalizations loc) {
+    return Card(
+      margin: EdgeInsets.only(top: 12, right: 12, left: 12),
+      child: Padding(
+        padding: const EdgeInsets.all(10),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Padding(
+              padding: const EdgeInsets.only(bottom: 8),
+              child: Text(
+                version.isLocal
+                    ? loc.versionsLocalChanges
+                    : loc.versionsVersionNumber(version.number),
+                style: TextStyle(fontWeight: FontWeight.bold, fontSize: 16),
+              ),
+            ),
+            if (!version.isLocal)
+              Padding(
+                padding: const EdgeInsets.only(bottom: 8.0),
+                child: Text(
+                  loc.versionsVersionMeta(
+                    version.user,
+                    DateFormat.yMMMMd().add_Hm().format(version.timestamp),
+                  ),
+                ),
+              ),
+            if (version.comment != null)
+              Padding(
+                padding: const EdgeInsets.only(bottom: 8.0),
+                child: Text(
+                  version.comment!,
+                  style: TextStyle(fontStyle: FontStyle.italic),
+                ),
+              ),
+            version.noTagChange
+                ? Text(loc.versionsNoTagChanges)
+                : _buildTable(version),
+          ],
+        ),
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final loc = AppLocalizations.of(context)!;
+
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(loc.versionsTitle(widget.fullRef)),
+        actions: [
+          IconButton(
+            icon: Icon(Icons.open_in_browser),
+            onPressed: () async => await launchUrl(widget.historyUrl,
+                mode: LaunchMode.externalApplication),
+          ),
+        ],
+      ),
+      body: Builder(
+        builder: (BuildContext context) {
+          if (error != null) {
+            return _buildError(loc);
+          }
+          if (history == null) {
+            return _buildLoader();
+          }
+
+          return ListView(
+            children: [
+              for (var version in history!.versions.reversed)
+                _buildCard(version, loc),
+              Container(height: 12), // bottom padding
```

Comment:
Please use `SizedBox` for empty spans :)

Reply:
Fixed in 8180d1e